src/app/data-planes/views/DataPlaneDetailView.vue

```diff
@@ -221,51 +233,58 @@
                   <template
                     v-else
                   >
-                    <ServiceTrafficGroup
-                      type="passthrough"
-                    >
-                      <ServiceTrafficCard
-                        :protocol="`passthrough`"
-                        :traffic="traffic.passthrough"
-                      >
-                        Non mesh traffic
-                      </ServiceTrafficCard>
-                    </ServiceTrafficGroup>
-                    <DataCollection
-                      v-slot="{ items }"
-                      :predicate="route.params.inactive ? undefined : (item) => ((item.protocol === 'tcp' ? item.tcp?.downstream_cx_rx_bytes_total : item.http?.downstream_rq_total) as (number | undefined) ?? 0) > 0"
-                      :items="traffic.outbounds"
+                    <!-- Outbounds for gateways report actual traffic on the upstream so we switch to upstream here for non-standard-->
+                    <template
+                      v-for="direction in [props.data.dataplane.networking.type !== 'standard' ? 'upstream' : 'downstream'] as const"
```

Comment:
**Suggestion**: I feel like this would be better solved by storing a new property on `networking` in the data layer.

Reply:
Just so I'm clear, do you mean the `direction` we should use to figure out the traffic?

Reply:
Yes.

Reply:
This will depend on whether its an inbound or outbound, which are different things so I don't think there will be a need for its own property. But I can re look at this in a follow up.